swap_meet/vendor.py

```diff
@@ -1,2 +1,95 @@
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else []
```

Comment:
Great job avoiding the side effects of having an empty list as the default value for the inventory argument. Great use of the ternary as well!